src/api/recepies.service.ts

```diff
@@ -0,0 +1,120 @@
+import { handleErrorSync } from '@stlib/utils';
+import { RecepieEntity, RecepieType } from '../db';
+import { CreateRecepieDto, UpdateRecepieDto } from './dto';
+import { Response, Request } from 'express';
+import { v4 as uuidv4 } from 'uuid';
+
+export class RecepiesService {
+  private recepieRespository: typeof RecepieEntity = RecepieEntity;
```

Comment:
what for is that?????, just use this entity, not need to create new variable

Reply:
fixed